pkg/cmd/backtest.go

```diff
@@ -438,6 +438,9 @@ var BacktestCmd = &cobra.Command{
 		}
 
 		runCtx, cancelRun := context.WithCancel(ctx)
+		for _, exK := range exchangeSources {
```

Comment:
when we have 2 exchange sources, these callbacks will be triggered twice?

Reply:
Yes, one for each source